app/views/categories/_create.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="card category-form">
```

Comment:
I am not sure how you get to this action but usually what you do is you make a form partial and you use it for both new and edit to render the form. The idea behind doing this is called DRY(Do not repeat yourself). It's common practice, here's a not so helpful wiki about it https://en.wikipedia.org/wiki/Don%27t_repeat_yourself